perllib/FixMyStreet/DB/ResultSet/Problem.pm

```diff
@@ -172,6 +172,10 @@ sub around_map {
             latitude => { '>=', $p{min_lat}, '<', $p{max_lat} },
             longitude => { '>=', $p{min_lon}, '<', $p{max_lon} },
     };
+
+    my $interval = $c->cobrand->report_age;
+    $q->{lastupdate} = { '>=', \"current_timestamp-'$interval'::interval" } unless
```

Comment:
Using `lastupdate` means very old reports could be shown if updated recently. Which is fine, but just to note as "Show older problems" could be taken to just mean the problems are old, not their latest activity.
And then, should this vary depending on what sort order `$p{order}` is set to? If you're sorting by newest, perhaps it should also be newest - last updated, lastupdate, and then oldest it probably doesn't make much sense to do it at all? Dunno.

Reply:
I see your point on `lastupdate` so will change that to `confirmed`. It's also probably more in line with what councils expect. 
Not sure that changing which reports show dependant on sort order is a very obvious behaviour.